### PR TITLE
:book: description upd, insctruction crt

### PR DESCRIPTION
**Pull request by Mirgalimova**

### CODE REVIEW

File: pos_journal_pin/doc/index.rst [hunk 1]
```diff
@@ -13,26 +13,26 @@ Configuration
...
 
 * Go to ``[[Point of Sale]]`` menu
-* Open POS session
+* Open a POS session
 
   * Click ``[Payment]``
-  * Select and click a payment method with enabled **Manager's Permission** option
-  * Select a manager from appeared list
-  * Enter it's password
+  * Select and click a payment method with enabled **POS Manager's Permission** option
```
Comment (KolushovAlexandr) [inline pos_journal_pin/doc/index.rst]: Please return it back. Here **Manager's Permission** is the field name so if we change it here then we have to change it in the code too

File: pos_journal_pin/doc/index.rst [hunk 2]
```diff
@@ -13,26 +13,26 @@ Configuration
...
+Set a PIN for a POS manager as it's explained in ``Confirm POS action by PIN`` module.
+As Administrator please follow next scenario:
 
 Journal
 -------
 
   * Open ``[[Point of Sale]] >> Payment Methods`` menu
   * Choose a journal or create one, be sure the journal is active in POS
-  * Activate **Manager's Permission** field
+  * Activate **POS Manager's Permission** field
```
Comment (KolushovAlexandr) [inline pos_journal_pin/doc/index.rst]: Please return it back. Here **Manager's Permission** is the field name so if we change it here then we have to change it in the code too

File: pos_journal_pin/doc/index.rst [hunk 3]
```diff
@@ -13,8 +13,8 @@ Configuration
 Manager's PIN
 -------------
 
-Set a PIN for a POS manager as it's explained in `Confirm POS action by PIN` module
-As Administrator do next:
+Set a PIN for a POS manager as it's explained in ``Confirm POS action by PIN`` module.
```
Comment (yelizariev) [inline pos_journal_pin/doc/index.rst]: Use a link to apps store please

File: pos_journal_pin/README.rst [hunk 4]
```diff
@@ -27,7 +25,7 @@ Maintainers
 
       To get a guaranteed support
       you are kindly requested to purchase the module
-      at `odoo apps store <https://apps.odoo.com/apps/modules/10.0//>`__.
+      at `odoo apps store <https://apps.odoo.com/apps/modules/10.0/journal_pin/>`__.
```
Comment (yelizariev) [inline pos_journal_pin/README.rst]: pos_journal_pin

File: pos_journal_pin/README.rst [hunk 5]
```diff
@@ -38,12 +36,12 @@ Further information
 
 Demo: http://runbot.it-projects.info/demo/pos-addons/10.0
 
-HTML Description: https://apps.odoo.com/apps/modules/10.0//
+HTML Description: https://apps.odoo.com/apps/modules/10.0/pos_pin/
```
Comment (yelizariev) [inline pos_journal_pin/README.rst]: pos_journal_pin

File: pos_journal_pin/doc/index.rst (hunk 3 above)
Comment (yelizariev) [reply pos_journal_pin/doc/index.rst]: STill no link :disappointed:

File: pos_journal_pin/README.rst (hunk 5 above)
Comment (Mirgalimova) [reply pos_journal_pin/README.rst]: image

Точно написать "pos_journal_pin" ?
Та ссылка, что сейчас стоит - рабочая, если поменять на "pos_journal_pin" будет 404)

File: pos_journal_pin/doc/index.rst (hunk 3 above)
Comment (Mirgalimova) [reply pos_journal_pin/doc/index.rst]: fixed

**Pull request merged**